actions/workflows/st2_bootstrap_test.yaml

```diff
@@ -0,0 +1,71 @@
+---
+version: '2.0'
+
+st2cd.st2_bootstrap_test:
+    type: direct
+    input:
+      - distro
+      - environment
+      - hostname
+      - instance_type 
+      - key_name
+      - keyfile
+      - role
+      - version
+      - st2_username: test
+      - st2_password: Ch@ngeMe
+    tasks:
+      create_vm:
+        # [105, 26]
+        action: st2cd.create_vm_role
+        input:
+          hostname: <% $.hostname %>
+          instance_type: <% $.instance_type %>
+          environment: <% $.environment %>
+          key_name: <% $.key_name %>
+          keyfile: <% $.keyfile %>
+          distro: <% $.distro %>
+          role: <% $.role %>
+
+        on-success:
+          - get_bootstrap_script
+      run_bootstrap_script:
+        # [105, 230]
+        action: core.remote_sudo
+        input:
+          hosts: <% $.hostname %>
+          cmd: bash /tmp/st2_bootstrap.sh
+          timeout: 600
+        on-success:
+          - end_to_end_tests
+      end_to_end_tests:
+        # [105, 332]
+        action: st2cd.e2e_bootstrap_tests
+        input:
+          hostname: <% $.hostname %>
+          st2_username: <% $.st2_username %>
+          st2_password: <% $.st2_password %>
+
+        on-success:
+          - destroy_vm
+      get_bootstrap_script:
```

Comment:
Can you move this below create_vm? It's causing my eyeballs to move up and down too much.